solidity/test/integration/FullFlow.test.ts

```diff
@@ -297,49 +301,66 @@ describeFn("Integration Test - Full flow", async () => {
     newDepositDustThreshold: BigNumberish,
     newDepositTxMaxFee: BigNumberish
   ) {
-    const currentDepositParameters = await bridge.depositParameters()
-    await bridge
+    console.log("duuupa")
+    await bridgeGovernance
       .connect(governance)
-      .updateDepositParameters(
-        newDepositDustThreshold,
-        currentDepositParameters.depositTreasuryFeeDivisor,
-        newDepositTxMaxFee
-      )
+      .beginDepositDustThresholdUpdate(newDepositDustThreshold)
+
+    await bridgeGovernance
+      .connect(governance)
+      .beginDepositTxMaxFeeUpdate(newDepositTxMaxFee)
+
+    await helpers.time.increaseTime(constants.governanceDelay)
+
+    await bridgeGovernance.connect(governance).finalizeDepositTxMaxFeeUpdate()
+
+    await bridgeGovernance
+      .connect(governance)
+      .finalizeDepositDustThresholdUpdate()
+
+    console.log("duuupa23")
+
+    // const currentDepositParameters = await bridge.depositParameters()
+    // await bridge
+    //   .connect(governance)
+    //   .updateDepositParameters(
+    //     newDepositDustThreshold,
+    //     currentDepositParameters.depositTreasuryFeeDivisor,
+    //     newDepositTxMaxFee
+    //   )
```

Comment:
```suggestion
```

Reply:
part of [40d1f5f](https://github.com/keep-network/tbtc-v2/pull/329/commits/40d1f5f0583534c0683a5f0b871608c216b85900)